management/src/main/java/org/ehcache/management/registry/DefaultCollectorService.java

```diff
@@ -113,6 +121,20 @@ public void cacheRemoved(String alias, Cache<?, ?> cache) {
         EhcacheNotification.CACHE_REMOVED.name()));
   }
 
+  private void cacheCleared(String alias) {
+    collector.onNotification(
+      new ContextualNotification(
+        configuration.getContext().with("cacheName", alias),
+        EhcacheNotification.CACHE_CLEARED.name()));
+  }
+
+  private void registerClearNotification(String alias, Cache<?, ?> cache) {
+    OperationStatistic<ClearOutcome> clear = StatsUtils.findOperationStatisticOnChildren(cache,
+      ClearOutcome.class, "clear");
+    clear.addDerivedStatistic(new OperationResultFilter<>(EnumSet.of(ClearOutcome.SUCCESS),
+      (long time, long latency) -> cacheCleared(alias)));
```

Comment:
You shouldn't need to type these parameters.

Reply:
ah right
```(long time, long latency) -> cacheCleared(alias)));```